plugins/woocommerce-blocks/assets/js/base/stores/add-to-cart.ts

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+import { store } from '@woocommerce/interactivity';
+import type { store as StoreType } from '@wordpress/interactivity'; // Todo: remove once we import from `@wordpress/interactivity`.
+
+export interface Store {
+	state: {
+		restUrl: string;
+		nonce: string;
+		cart: {
+			items: {
+				key?: string;
+				id: number;
+				quantity: number;
+			}[];
+		};
+	};
+	actions: {
+		addToCart: ( id: number, quantity: number ) => void;
+		refreshCart: () => void;
+	};
+}
+
+let pendingRefresh = false;
+
+// Todo: Remove the type cast once we import from `@wordpress/interactivity`.
+export const { state, actions } = ( store as typeof StoreType )< Store >(
+	'woocommerce',
+	{
+		actions: {
+			// Question: Should we name this `addCartItem` to match the Store API naming?
```

Comment:
Question: Should we name this `addCartItem` to match the Store API naming?

It would be good to take a look at the Store API to see what other actions we need to create and if it makes sense for the names of those actions to correspond more closely.

Reply:
For now, I've renamed it to `addCartItem` to try to make it more similar to the Store API, but we should keep an eye on this as we progress.